### PR TITLE
Fixes for SQL Replication Cmdlets and tests

### PR DESCRIPTION
**Pull request by dominiqa**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dominiqa__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/ResourceManager/Resources/Commands.Resources/AzureResourceManager.psd1 [hunk 1]
```diff
@@ -55,7 +55,8 @@ ScriptsToProcess = @()
   
 # Type files (.ps1xml) to be loaded when importing this module  
 TypesToProcess = @(  
-    '.\Compute\Microsoft.Azure.Commands.Compute.Types.ps1xml'  
+    '.\Compute\Microsoft.Azure.Commands.Compute.Types.ps1xml',
+	'.\Sql\Microsoft.Azure.Commands.Sql.Types.ps1xml'
```
Comment (markcowl) [inline src/ResourceManager/Resources/Commands.Resources/AzureResourceManager.psd1]: correct spacing, pleaser

File: setup/azurecmdfiles.wxi [hunk 2]
```diff
@@ -578,6 +578,15 @@
                         <Component Id="cmpF08FE4050A937C6DD895E92ABAE2A275" Guid="*">
                             <File Id="fil7EAD0480A7385B639C6A89FD485C13D0" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureBatch\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
+                        <Component Id="cmp1D1A78885A81C3FE9B8F9FBFAD6EB010" Guid="*">
```
Comment (markcowl) [inline setup/azurecmdfiles.wxi]: this is too many changes in the wix file - the only change that should be occuring is the additon of your ps1xml file.  Please clean your directory, rerun, and regenerate the msi.

File: src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseReplicationTests.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
+{
+    public class DatabaseReplicationTests : SqlTestsBase
+    {
+        [Fact]
+        [Trait(Category.Sql, Category.CheckIn)]
```
Comment (markcowl) [inline src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseReplicationTests.cs]: None of these will be run as part of check-in.  Please change the trait to

```c#
[Trait(Category.AcceptanceTyoe, Category.CheckIn)]
```

for those tests that should be run as check-in tests.

**Comment by markcowl** [comment]
@dominiqa please respond to feedback

File: setup/azurecmdfiles.wxi (hunk 2 above)
Comment (dominiqa) [inline setup/azurecmdfiles.wxi]: Tried to clean this up, but the BuildInstaller.ps1 still generated a bunch of changes

**Comment by dominiqa** [comment]
All feedback addressed

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by ogail** [comment]
One demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/151/console

**Comment by OJDUDE** [comment]
I signed the CLA. Is there anything else blocking the code merge?

**Pull request merged**